hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/bucket/BucketCache.java

```diff
@@ -429,7 +429,9 @@ private void cacheBlockWithWaitInternal(BlockCacheKey cacheKey, Cacheable cached
     if (!cacheEnabled) {
       return;
     }
-    LOG.trace("Caching key={}, item={}", cacheKey, cachedItem);
+    if (LOG.isTraceEnabled()) {
```

Comment:
Don't need this if LOG.isTraceEnabled when using this logging form with the '{}' (Internally it does this test).